yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -367,8 +381,8 @@ func init() {
 	importDataFileCmd.Flags().StringVar(&fileFormat, "format", "csv",
 		fmt.Sprintf("supported data file types: %v", supportedFileFormats))
 
-	importDataFileCmd.Flags().StringVar(&delimiter, "delimiter", ",",
-		"character used as delimiter in rows of the table(s)")
+	importDataFileCmd.Flags().StringVar(&delimiter, "delimiter", "",
+		`character used as delimiter in rows of the table(s)(default for csv: ',' andgit  for text: '\t')`)
```

Comment:
`git   `???

Reduce the number of words to convey the information:
(DEFAULT: comma for CSV and tab for TEXT format)